Source/ChatAutocompleteIntegrator.lua

```diff
@@ -130,9 +131,9 @@ function ChatAutocompleteIntegrator:_OnItemSearchComplete(editBox, items, search
 
   -- Since this is received asynchronously, discard the result if it has become irrelevant
   if util.IsNilOrEmpty(searchTerm) or searchTerm:find(searchInfo.searchTerm, nil, true) ~= 1 then
+    util.PrettyPrint('Item search DISCARDED' + searchTerm)
```

Comment:
Is this a print a left-over from debugging or does it fulfill another purpose?

Reply:
Most likely left over debugging from trying to trigger this.